.prettier.config.js

```diff
@@ -1,35 +1,35 @@
-'use strict';
+"use strict";
 
 module.exports = {
-  arrowParens: 'always',
+  arrowParens: "always",
   bracketSpacing: true,
-  endOfLine: 'lf',
+  endOfLine: "lf",
   printWidth: 80,
-  quoteProps: 'consistent',
+  quoteProps: "consistent",
   semi: true,
   singleQuote: true,
   tabWidth: 2,
-  trailingComma: 'es5',
+  trailingComma: "es5",
```

Comment:
Codacy has a fix for the issue: [Replace `"es5"` with `'es5'`](https://app.codacy.com/gh/openinf/openinf.github.io/pullRequest?prid=9929905)
```suggestion
  trailingComma: 'es5',
```